packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellContainer.tsx

```diff
@@ -18,6 +23,28 @@ import { RecordTableCellEditMode } from './RecordTableCellEditMode';
 
 import styles from './RecordTableCellContainer.module.css';
 
+const StyledSkeletonContainer = styled.div`
+  padding-left: ${({ theme }) => theme.spacing(2)};
+  padding-right: ${({ theme }) => theme.spacing(1)};
+`;
+
+const StyledRecordTableCellContainerLoader = ({
```

Comment:
Could you extract all of this logic outside of RecordTableCellContainer ? 

Because RecordTableCellContainer is a very optimized component used for displaying FieldDisplay component fast.

Here we just want to fill a table with skeletons, so we might as well put this logic in another RecordTableCellLoading and use it directly in the new RecordTableBodyLoading that should also be extracted.

Reply:
Looking into this @lucasbordeau